android/src/main/java/com/appgoalz/rnjwplayer/RNJWPlayerView.java

```diff
@@ -7,13 +7,16 @@
 import android.content.Context;
 import android.content.Intent;
 import android.content.ServiceConnection;
+import android.content.pm.Activity;
```

Comment:
```suggestion
```

this API is deprecated, and will result in compile error in my case

Reply:
Yup getting the same compilation error. Forgot to mention this one... :/